extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -2332,6 +2332,16 @@ public static Token fromAudience(String... audience) {
          */
         public Optional<String> decryptionKeyLocation = Optional.empty();
 
+        /**
+         * Decrypt ID token.
+         */
+        Optional<Boolean> decryptIdToken = Optional.empty();
```

Comment:
```suggestion
        /**
         * Decrypt ID token.
         */
       boolean decryptIdToken;
```

Reply:
Hi @gastaldi, there is a reason I did this way, see `point 2` in the PR description, otherwise, it would definitely make no sense, good catch.

Does that point in the description make sense ?

Reply:
@gastaldi, so I'd like to ensure that the current behaviour where the decryption-key-location is set causes an ID token decryption attempt is retained, to avoid breaking changes, but also, I'd like to let users decrypt access tokens only but not ID tokens if that key location is set